<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workWindow</class>
 <widget class="QMainWindow" name="workWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>同游主界面</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_6"/>
    <addaction name="action_5"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>新增行程</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>编辑行程</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>浏览行程</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>导入或导出</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>查询行程</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
